### PR TITLE
Community: added audio-parser RemoteFasterWhisperParser in audio.py

### PR DESCRIPTION
**Pull request by lfenzo**
## Description

This PR introduces a new audio parser to Langchain that allows interaction with [Faster Whisper](https://github.com/SYSTRAN/faster-whisper) running in a remote server. This enhancement addresses a significant issue with the current Faster Whisper integration, which requires the Langchain process to have all necessary GPU drivers and dependencies installed locally. In microservice-based architectures, this leads to heavy and redundant dependencies in the Langchain container, making deployment more complex and resource-intensive.

## Solution

This PR solves this issue by introducing a remote audio parser that can communicate with [faster-whisper-server](https://github.com/fedirz/faster-whisper-server) running in a dedicated container. This allows Langchain to delegate audio transcription tasks to this remote server, eliminating the need for GPU dependencies within the Langchain container itself reducing its dependency footprint.

**The implementation relies on Python's built-in `subprocess` module to handle communication with the remote server, ensuring that no extra dependencies are introduced in Langchain.**

-----------------

cc.: @eyurtsev

### CODE REVIEW

**Comment by lfenzo** [comment]
@baskaryan

**Comment by lfenzo** [comment]
@efriis

**Comment by efriis** [comment]
Marking as stale and still in need of support, per the [review process](https://python.langchain.com/docs/contributing/reference/review_process/)

File: libs/community/langchain_community/document_loaders/parsers/audio.py [hunk 1]
```diff
@@ -690,3 +691,143 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
...
+            file_bytes (io.BytesIO): The audio file data in bytes format.
+
+        Returns:
+            dict[str, str]: A dictionary containing the transcription result.
+
+        Raises:
+            RuntimeError: If the transcription process fails.
+        """
+
+        process = subprocess.Popen(
```
Comment (efriis) [inline libs/community/langchain_community/document_loaders/parsers/audio.py:780]: unfortunately this isn't acceptable for security reasons, so will close!

**Pull request closed without merging**